src/cc2olx/models.py

```diff
@@ -123,7 +171,7 @@ def normalize(self):
             course_root = course_root[0]
         if not course_root:
             return
-        sections = course_root.get("children", [])
+        sections = self.process_canvas_cc(course_root.get("children", []))
```

Comment:
Should we run this in all cases, or only run this transformation for canvas flavored common cartridges?
We can check if `course_settings/canvas_export.txt` exists to determine if this is a canvas flavored CC.
Canvas [writes this file to determine if it's from a canvas export](https://github.com/instructure/canvas-lms/blob/8a3f0249376ca246106f6505063e68b5829fcfac/lib/cc/canvas_resource.rb#L90).

Reply:
@kaizoku, Thanks for the suggestion. Yeah this approach would be better. I've updated my PR.